src/app/trainer/dashboard/edit/page.jsx

```
@@ -0,0 +1,255 @@
+            {/* Desktop tabs - hidden on mobile */}
+            <div className="hidden md:flex overflow-x-auto gap-1 mb-6 pb-1 scrollbar-hide border-b border-[rgba(255,255,255,0.1)]">
+              {TRAINER_SECTIONS.map((section) => (
+                <button
+                  key={section.id}
+                  onClick={() => setActiveSection(section.id)}
+                  className={`flex items-center gap-2 whitespace-nowrap px-4 py-3 transition-all ${
+                    activeSection === section.id
+                      ? "text-[#FF6B00] border-b-2 border-[#FF6B00]"
+                      : "text-gray-400"
+                  }`}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)